components/eamxx/src/physics/shoc/impl/shoc_pblintd_impl.hpp

```diff
@@ -78,7 +77,7 @@ void Functions<S,D>::pblintd(
 
   // Initialize
   bool check = true;
-  s_rino(nlev-1) = 0;
+  Kokkos::deep_copy(rino, 0);
```

Comment:
I thought deep_copy was not allowed on device... You may instead do something like
```c++
#ifndef NDEBUG
for (size_t i=0; i<rino.size(); ++i)
  rino(i)=0;
#endif
```

Reply:
Ah, yeah I tested this on mappy.